src/common/footer/Footer.tsx

```diff
@@ -33,6 +33,7 @@ const Footer = () => {
       title={t('appName')}
     >
       <HDSFooter.Utilities>
+        <HDSFooter.Link as={Link} to="/about" label={t('footer.about')} />
```

Comment:
According to ticket description, this should be located next to Accessibility document and Cookie settings. 🤔 Moving this inside `<HDSFooter.Base />` should do the trick